src/CoreWCF.Http/src/CoreWCF/HttpTransportSecurity.cs

```diff
@@ -22,6 +22,8 @@ public HttpTransportSecurity()
             _extendedProtectionPolicy = ChannelBindingUtility.DefaultPolicy;
         }
 
+        public string CustomAuthenticationScheme { get; set; }
+
```

Comment:
I don't think we need to specify a scheme if we are just leaving it up to asp.net core. I had a look at the the authentication code and we can pass null as the scheme which means look up the default configured authentication scheme.

Reply:
done